.eslintrc

```diff
@@ -150,7 +151,7 @@ rules:
     no-undefined: 0                # disallow use of undefined variable
     no-unused-vars: 2              # disallow declaration of variables that are not used in
                                    #    the code
-    no-use-before-define: 2        # disallow use of variables before they are defined
+    no-use-before-define: 1        # disallow use of variables before they are defined
```

Comment:
I think adding the es6 env made it so eslint started checking this rule... there are 2 instances in the code base from before this PR that don't follow this rule.